### DISCUSSION TITLE
Ai Trends

### DISCUSSION THREAD

**Discussion by TimothyMichaelGould** [janhq/jan#1421] [General]
Love where this is headed. As someone who uses Ai to capture concepts I think of, Ai, has a long way to go to understand how we communicate ideas but it's getting there. :-) Much of what I want to capture in ideas entails deep thought processes as well as a vision that goes beyond a simple answer. Often times, Ai only understands a limited aspect about complex subjects before it loses the ability to apprehend a vision handed to it. I do enjoy when it delivers though.

**Comment by PiotrWarzachowski** [comment]
I wanted to contribute a bit by making this open source project, where I used a ChatGPT to summarize PDF documents, you can contribute to change it to jan if you want, here is my repo : https://github.com/PiotrWarzachowski/DocMind Feel free to check it out, its public on vercel. You can leave a star ⭐ if you want 💓. Have a good year
